feat: show sidebar by meeting summary

nickoferrall opened the pull request:
Fix https://github.com/ParabolInc/parabol/issues/8749

Demo: https://www.loom.com/share/cc327831b7614612b08c50acc2059cf7

### To test

- [ ] See the sidebar in the meeting summary on desktop but not mobile
- [ ] The "Go to Team Dashboard" button doesn't appear on desktop but it does appear on mobile
- [ ] Click download to PDF and see that the sidebar doesn't show-up in the PDF
- [ ] The sidebar doesn't show up in the email summary

acressall commented:
_these are questions, not blockers_

- Is it much effort to add the top bar as well, for consistency? Then the open/closed, desktop/mobile behaviour can be the same too
- If I go to a summary from the history view, will the navigation be there too?

igorlesnenko reviewed (changes requested):
I'm seeing one small issue: https://www.loom.com/share/d76c77e32d5f41a2afe63a57bc1e625e

nickoferrall commented:
> I'm seeing one small issue:

@igorlesnenko oh nice! Very helpful having so many teams. Updated!


> Is it much effort to add the top bar as well

I've added the top bar on desktop 👍  On mobile, the top bar and the sidebar are hidden, so it's the same as it is today

![Screenshot 2023-09-21 at 16 44 16](https://github.com/ParabolInc/parabol/assets/39854876/166cfa78-337d-47d6-ab6a-4ea69e7e1f16)


> If I go to a summary from the history view, will the navigation be there too?

Yep - as in if you navigate to the summary from the history view or anywhere else in the app, the summary will look & act the same

acressall commented:
There's one thing we forgot -- where it says the team name, under the retro title, it should link to the team dashboard. That way you can still get there from email or mobile

igorlesnenko reviewed (approved):
Found one more issue: https://www.loom.com/share/0c1b9cb60fdd47c4bffabc0e0e92288a

igorlesnenko reviewed (changes requested):
Found one more issue: https://www.loom.com/share/0c1b9cb60fdd47c4bffabc0e0e92288a

igorlesnenko commented on packages/client/components/Dashboard/DashSidebar.tsx:
```diff
@@ -154,6 +154,7 @@ const DashSidebar = (props: Props) => {
           <DashHR />
           <NavMain>
             <NavList viewer={viewer} />
+            <NavList viewer={viewer} />
```
how does it work? We now have 2 same components?

nickoferrall replied:
I added that to reproduce the UI issue you mentioned. Thanks, removed!

nickoferrall commented:
@acressall I've made the team name clickable. We do still have the "Go to Team Dashboard" CTA on mobile. Let me know if you were thinking we would remove this. The [design](https://www.figma.com/file/HQkcXyUtEIhDH27GdchRxr/view---Meeting-domino?type=design&node-id=21-449&mode=design&t=UVbJ1Nf3eOdzBxuS-0) I see has the CTA, but it was made before going with the sidebar idea.

@igorlesnenko I can't reproduce the iPad UI scrollbar issue. Would you mind checking again and seeing if it behaves differently to master?

Loom demo: https://www.figma.com/file/HQkcXyUtEIhDH27GdchRxr/view---Meeting-domino?type=design&node-id=21-449&mode=design&t=UVbJ1Nf3eOdzBxuS-0

igorlesnenko commented:
@nickoferrall You linked figma instead of zoom, can you link zoom video? Checking now

igorlesnenko commented:
@nickoferrall oh I see now same issue on master, so probably all good

Dschoordsch commented on packages/client/modules/summary/components/NewMeetingSummary.tsx:
```diff
@@ -19,8 +21,10 @@ interface Props {
 
 const query = graphql`
-  query NewMeetingSummaryQuery($meetingId: ID!) {
+  query NewMeetingSummaryQuery($meetingId: ID!, $first: Int!, $after: DateTime) {
```
+1 I know it's out of scope of this PR, but having generic `$first` is not a good name for a variable here, because it requires some search to find out it's about notifications. It would be nicer to have `$notificationsFirst` and `$notificationsAfter` here, but that would require changes in a lot of other places.

Dschoordsch commented:
One note, not sure if this is intentional, but we now show the sidebar in the demo summary as well
<img width="1728" alt="Screenshot 2023-09-25 at 14 58 57" src="https://github.com/ParabolInc/parabol/assets/7331043/e3aa506a-a7ca-4ce1-86e2-1eb7e11612ef">
All links will just redirect to the login dialog, so that's not an issue and I actually think it's nice to see some more preview of the app later. But it would be even nicer, if we'd redirect to the register dialog instead, but that's probably a bit harder to achieve.

nickoferrall commented on packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/SummarySheet.tsx:
```diff
@@ -88,6 +90,7 @@ const SummarySheet = (props: Props) => {
     `,
     meetingRef
   )
+  const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
```
I've removed useBreakpoint and hidden this using tailwind 👍
